### DISCUSSION TITLE
Import from postman doesn't import Tests/PreRequest scripts

### DISCUSSION THREAD

**Discussion by ifnotnithya** [hoppscotch/hoppscotch#3267] [Q&A] [answered]
I am looking at importing few collections from postman.
This is not importing the Pre Request script or Tests.

Is this not supported ?

Thanks

**Comment by jbristol-trav** [comment]
We are also facing issues with this as we migrate from Postman

**Comment by jamesgeorge007** [comment, marked as answer]
Hi, we’ve added support for importing scripts from Postman collections (see #5417) in the [latest release](https://github.com/hoppscotch/hoppscotch/releases/tag/2025.10.0), as part of ongoing scripting improvements based on RFC #5221. Please note that this feature is experimental and requires opting into the `Experimental scripting sandbox` option via settings.

<img width="1485" height="529" alt="Image" src="https://github.com/user-attachments/assets/a7e9ec55-af17-4763-8062-c9bee9891b71" />

We welcome your feedback, which will help shape the next generation of Hoppscotch scripting. Feel free to share it via the RFC.
